proto/options.proto

```diff
@@ -322,6 +322,9 @@ message Costing {
     bool exclude_tolls = 89;
     bool exclude_highways = 90;
     bool exclude_ferries = 91;
+    oneof has_ignore_constructions {
+      bool ignore_constructions = 92;
+    }
```

Comment:
default is false so you dont need `oneof`
```suggestion
    bool ignore_constructions = 92;
```

Reply:
Done!